addons/project/views/project_views.xml

```diff
@@ -1669,6 +1669,18 @@
             </field>
         </record>
 
+        <record id="open_view_my_tasks_list_view" model="ir.ui.view">
+            <field name="name">Name</field>
```

Comment:
Just a small last request, this view should be called something different than "Name". :innocent: 
You can just copy the id and replace the `_` with `.` if you don't have any inspiration
```suggestion
            <field name="name">open.view.my.tasks.list.view</field>
```